templates/ovndbcluster/bin/setup.sh

```diff
@@ -127,6 +127,8 @@ if [[ "$(hostname)" == "{{ .SERVICE_NAME }}-0" ]]; then
     # The daemon is no longer needed, kill it
     kill $(cat $OVN_RUNDIR/ovn-${DB_TYPE}ctl.pid)
     unset OVN_${DB_TYPE^^}_DAEMON
+
+    trap wait_for_ovsdb_tool EXIT
```

Comment:
I think this should happen before the tool is started, line 91?

Reply:
And once ovsdb-tool is done, we can probably remove the trap with `trap - EXIT`

Reply:
> I think this should happen before the tool is started, line 91?
Done

> And once ovsdb-tool is done, we can probably remove the trap with trap - EXIT
Done